security/subsystem/src/main/java/org/jboss/as/security/LegacySupport.java

```diff
@@ -328,6 +328,7 @@ private ModelNode getNoTextValueTypeDescription(final ModelNode parent) {
         static {
             final ParametersValidator delegate = new ParametersValidator();
             delegate.registerValidator(CODE, new StringLengthValidator(1));
+            delegate.registerValidator(Constants.MODULE, new StringLengthValidator(1, true));
```

Comment:
OK, let me revert my changes in `LegacySupport`. Obviously, my intention was to add the handling of the `module` attribute everywhere where login module had it.